### PR TITLE
feat: Restrict minute-resolution sessions to a 7h time window

### PR DESCRIPTION
**Pull request by Swatinem**
This also removes the config switch, always enabling minute-resolution sessions unconditionally.

### CODE REVIEW

File: tests/datasets/test_sessions_processing.py [hunk 1]
```diff
@@ -85,6 +84,12 @@ def query_runner(
             "selected_columns": ["sessions", "bucketed_started"],
             "groupby": ["bucketed_started"],
             "granularity": 60,
+            # XXX: using `conditions` here, as `from_date`/`to_date` is not being
```
Comment (evanh) [inline tests/datasets/test_sessions_processing.py:87]: from_date and to_date are parsed by query extensions. See this function: https://github.com/getsentry/snuba/blob/master/snuba/request/schema.py#L208

Your test below is only running the processors.

File: snuba/datasets/entities/sessions.py [hunk 2]
```diff
@@ -218,10 +218,6 @@ def select_storage(
             },
         )
 
-        allow_subhour_sessions = state.get_config("allow_subhour_sessions", 0)
```
Comment (evanh) [inline snuba/datasets/entities/sessions.py:221]: What's the impact of this? Is there anyone for whom this is not already available?

Comment (Swatinem) [reply snuba/datasets/entities/sessions.py:221]: This was added just as a failsafe back then. It was never turned on in production, but the metrics that are being tracked in the lines above gave some insight into which storage would have been chosen.

**Review by evanh — APPROVED** [review]

**Comment by fpacifici** [comment]
What is your product rollout plan going to be? We need to be sure that this does not put too much load on the sessions_raw table. I would suggest we do it progressively.

**Comment by Swatinem** [comment]
Sentry org and superusers so far, we can evaluate as we go, and probably discuss pricing for that feature as well.

**Pull request merged**